lib/pf/person.pm

```diff
@@ -38,7 +38,7 @@ BEGIN {
         person_violations
         person_custom_search
     );
-    @EXPORT_OK = qw( $PID_RE );
+    @EXPORT_OK = qw( $PID_RE @FIELDS );
```

Comment:
No need to export '@FIELDS' since you use the complete package name each time you use it.
We also try to avoid exporting only for exporting :)